src/core/nginx.go

```diff
@@ -160,13 +155,21 @@ func (n *NginxBinaryType) GetNginxDetailsByID(nginxID string) *proto.NginxDetail
 	return n.nginxDetailsMap[nginxID]
 }
 
-func (n *NginxBinaryType) GetNginxDetailsFromProcess(nginxProcess Process) *proto.NginxDetails {
+func (n *NginxBinaryType) sanitizeProcessPath(nginxProcess *Process) bool {
 	defaulted := false
 	if nginxProcess.Path == "" {
 		nginxProcess.Path = defaultToNginxCommandForProcessPath()
 		defaulted = true
 	}
+	if strings.Contains(nginxProcess.Path, execDeleted) {
+		log.Infof("nginx was upgraded (process), using new info")
```

Comment:
Could you remove this log line or change the log level here from info to debug? Its being repeated multiple times in the log file for the agent